pyproject.toml

```diff
@@ -5,6 +5,36 @@ description = "Django app to integrate with OpenAI Assistants API"
 authors = ["Vinta Software <contact@vinta.com.br>"]
 license = "MIT"
 readme = "README.md"
+keywords = [
+    "ai-agent",
+    "ai-assistant",
+    "ai",
+    "chatgpt",
+    "django",
+    "gpt",
+    "langchain",
+    "openai",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Build Tools",
```

Comment:
Should be something other than build tools here, I guess?

Reply:
Perhaps `Topic :: Scientific/Engineering :: Artificial Intelligence`? The full list is at: https://pypi.org/classifiers/

Reply:
- Environment :: Web Environment
- Framework :: Django
- Software Development :: Libraries :: Python Modules
- Topic :: Scientific/Engineering :: Artificial Intelligence